recipes/conda-wrappers/meta.yaml

```diff
@@ -0,0 +1,28 @@
+{% set version = "1.0.0" %}
+
+package:
+  name: conda-wrappers
+  version: {{ version }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - python
+  run:
+    - python
+    - exec-wrappers 1*
```

Comment:
> Probably should be 1.*

Are you sure, on previous conda-build version the dot had to be omitted, otherwise it never was able to match any version

> Also why this particular pinning?

I intend to respect semantic version with ```exec-wrappers``` so that new versions will be backward compatible. As this package depends on the first version, it does not really matter which one is exactly. It could be anything ```>= 1.0.0 and < 2.0.0```